src/auth/Auth.ts

```diff
@@ -252,22 +252,19 @@ export class Auth {
       });
     }
 
-    return this.refreshRequest.then(
-      async ({ data }) => {
+    return this.refreshRequest
+      .then(async ({ data }) => {
         await this.setToken(data);
         await this.handleExpired(data);
         if (this.authCallback) {
-          return this.authCallback(AuthCallbackActions.Refreshed, data, this);
+          this.authCallback(AuthCallbackActions.Refreshed, data, this);
```

Comment:
@gamezinfantes , you are modifying the previous behaviour, the authCallback might change the token value (although it's not currently the case), you might want to keep the returned value, sth like:

```
let newData: ReturnToken | undefined = data;
if (this.authCallback) {
  newData = await this.authCallback( ... );
}
...
return newData;